appveyor.yml

```diff
@@ -0,0 +1,9 @@
+version: '4.5.4-SNAPSHOT+AppVeyor.{build}'
+os: Windows Server 2012
+build_script:
+  - mvn compile -q --batch-mode 
+test_script:
+  - mvn install -q -Dantlr-python2-python="C:\Python27\python.exe" -Dantlr-python3-python="C:\Python35\python.exe" -Dantlr-javascript-nodejs="C:\Program Files (x86)\nodejs\node.exe" --batch-mode
```

Comment:
💭 I have no experience with the dependencies listed here (w.r.t. AppVeyor). Do you know if they come pre-installed on AppVeyor?